v2.0/paths/task_assignments__delete_task_assignments_id.yml

```diff
@@ -19,3 +19,6 @@ responses:
     description: |-
       An empty response will be returned when the task
       assignment was successfully deleted.
+  404: 
```

Comment:
Does this response also include a body? 

Can we make the description match the format “Returns an error if…” as per https://github.com/box/box-openapi/blob/master/v2.0/paths/chunked_uploads__post_files_id_upload_sessions.yml#L51?

Reply:
All of them look like this:
```{
    "type": "error",
    "status": 404,
    "code": "not_found",
    "help_url": "http://developers.box.com/docs/#errors",
    "message": "Authorization Failed",
    "request_id": "0cuds929ouuka1qglngdd07ac9c"
}```

I'll update the descriptions!

Reply:
Ah, in this case, for any of these responses just make them have a content of `ClientError`. See other 4XX errors for examples.

Reply:
So for anything that returns a generic client object, just use this syntax please. 

```yaml
4XX:
    description: |-
      Returns an error if ...
    content:
      application/json:
        schema:
          $ref: '#/components/schemas/ClientError'
```